resources/views/login.blade.php

```diff
@@ -0,0 +1,28 @@
+@extends('master')
+@section('content')
+    <form action="login" class="custom-form" method="POST">
```

Comment:
action sử dụng route
```suggestion
    <form action="{{ route('name-route') }}" class="custom-form" method="POST">
```